src/client/sandbox/event/listeners.js

```diff
@@ -296,4 +296,18 @@ export default class Listeners extends EventEmitter {
             eventListeningInfo.outerHandlersWrapper = wrapper;
         }
     }
+
+    getEventListeners (el, event) {
+        const eventCtx = this.listeningCtx.getEventCtx(el, event);
+
+        if (!eventCtx)
+            return null;
```

Comment:
I don't think so. It opens a way to fail in the code.
In fact we have two states: 
1) we don't listen element, so we don't know does it have handlers
2) we listen it and return results.

Imagine I'll use it in `testcafe` after some time for some element. And I get `[]` every time. How can I be sure is it element without event listeners or it's just not listened by hammerhead (and I should call `initElementListening`)?